entrypoint.sh

```diff
@@ -48,24 +48,26 @@ echo $NEWRELIC_KEY | sudo newrelic-install install
 # Configure PHP
 j2 /usr/local/etc/php/php.ini.j2 > /usr/local/etc/php/php.ini 
 
-
 # Configure SSMTP
 j2 /etc/ssmtp/ssmtp.conf.j2 > /etc/ssmtp/ssmtp.conf
 
-supervisorctl restart php-fpm
-supervisorctl restart nginx
-
 # Put Zed host IP to /etc/hosts file:
 echo "127.0.0.1	$ZED_HOST" >> /etc/hosts
 
 if [ -f /data/initialize ]
 then
     /setup_suite.sh
+    # Disable maintenance mode to validate LetsEncrypt certificates
+    test -f /maintenance_on.flag && rm /maintenance_on.flag
+    /setup_ssl.sh ${YVES_HOST//www./} $(curl http://169.254.169.254/latest/meta-data/public-ipv4 -s)
```

Comment:
`curl http://169.254.169.254/latest/meta-data/public-ipv4` couldn't be used out of AWS, should be more global solution.

Reply:
https://api.ipify.org/
???

Reply:
http://checkip.amazonaws.com/